### PR TITLE
Update screenshots + add PWA install guide

### PR DESCRIPTION
**Pull request by xepozz**

### CODE REVIEW

File: docs/guide/en/pwa_install.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# How to use Yii Dev Panel as PWA
+
+PWA – Power Web Application. Kind of web applications that may be installed into a system.
```
Comment (samdark) [inline docs/guide/en/pwa_install.md:3]:
```suggestion
PWA – Progressive Web Application is a web application that may be installed into a system and behaves as a native app.
```

File: docs/guide/en/pwa_install.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# How to use Yii Dev Panel as PWA
+
+PWA – Power Web Application. Kind of web applications that may be installed into a system. 
+
+Once install application can be connected to different application just by switching the base url from settings.
```
Comment (samdark) [inline docs/guide/en/pwa_install.md:5]:
```suggestion
Once installed, the application can be connected to different backend by switching the base URL from settings.
```

File: packages/yii-dev-panel/README.md [hunk 3]
```diff
@@ -152,6 +152,32 @@ npm start
 
 `yii-dev-panel` is now serving on http://localhost:3000
 
-## Screenshot
-
-![Screenshot](docs/screenshot.png)
+## Screenshots
+
+
+<details>
```
Comment (samdark) [inline packages/yii-dev-panel/README.md:158]: Do we need to hide these under spoilers?

**Review by samdark — APPROVED** [review]

File: packages/yii-dev-panel/README.md (hunk 3 above)
Comment (xepozz) [reply packages/yii-dev-panel/README.md:158]: There are lots of images, I'd hide them to make page simple

**Pull request merged**